lib/common/tests/Dom/DocumentTest.php

```diff
@@ -806,4 +860,21 @@ public function testGetElementIdOnPreexistingIds()
 
         $this->assertEquals('some-prefix-3', $dom->getElementId($element, 'some-prefix'));
     }
+
+    /**
+     * Call a private method as if it was public.
+     *
+     * @param object|string $object     Object instance or class string to call the method of.
+     * @param string        $methodName Name of the method to call.
+     * @param array         $args       Optional. Array of arguments to pass to the method.
+     * @return mixed Return value of the method call.
+     * @throws ReflectionException If the object could not be reflected upon.
+     */
+    private function callPrivateMethod($object, $methodName, $args = [])
```

Comment:
I assume you've inlined this here as it is the only occurrence in this library so far, with the intent of refactoring later if a need arises.

Reply:
Removed